doc/opf/powermodels.rst

```diff
@@ -6,26 +6,26 @@ Optimization with PandaModels.jl
 Installation
```

Comment:
can you please add a short introduction like below:

"Introduction:  PandaModels.jl (**panda**power + Power**Models**.jl) is a developed Julia package enabling the connection of pandapower and PowerModels in a stable and functional way. Except for calling the implemented optimization models in PowerModels, users can create custom optimization models with PandaModels. Presently, users can solve some reactive power optimization problems with PandaModels. "

thank you, I will further improve the doc.